### PR TITLE
Bump version to 4.6.17

### LINKED ISSUE #1013: Bump PHP Modules and Release: php-buildpack (03-14)
Release reminder for php-buildpack

The ideal release date window for this buildpack starts on: 03-14 and ends on 03-19.

* Bump PHP modules. See [doc](https://github.com/cloudfoundry/buildpacks-ci/tree/master/scripts/php-modules#pre-buildpack-release-task)
* See [diff from latest version](https://github.com/cloudfoundry/php-buildpack/compare/v4.6.16..develop) and validate if a release is required.
* Make sure the latest commit on  has passed tests on the [CI](https://buildpacks.ci.cf-app.com/teams/main/pipelines/php-buildpack)
* Refer [release instructions](https://github.com/pivotal-cf/tanzu-buildpacks/wiki/Releasing-CF-Buildpacks). (private link)

### PR DESCRIPTION
**Pull request by sophiewigmore**
resolves #1013

### CODE REVIEW

**Comment by sophiewigmore** [comment]
wait until pipelines are green to merge

**Review by brayanhenao — APPROVED** [review]

**Pull request merged**
